pod/perlfunc.pod

```diff
@@ -3388,20 +3401,20 @@ For example, to glob filenames that have an C<e> followed by a space
 followed by an C<f>, use one of:
 
     my @spacies = <"*e f*">;
-    my @spacies = glob '"*e f*"';
-    my @spacies = glob q("*e f*");
+    my @spacies = glob('"*e f*"');
```

Comment:
I don't think these need to be changed.

Reply:
This one I disagree on... I think for consistency all calls should be the same. These are a little harder to read maybe, but they're not different than any of the other glob calls where we used parens.